### DISCUSSION TITLE
提取深层特征的宽和高

### DISCUSSION THREAD

**Discussion by dun-123-tudou** [STVIR/pysot#393] [General]
作者你好，为什么主干网络像res50，在提取深层特征时的H，W，没有进一步缩小？呢（和目标检测中不太一样，请问这样有什么好处呢？）

**Comment by ZhiyuanChen** [comment]
> 作者你好，为什么主干网络像res50，在提取深层特征时的H，W，没有进一步缩小？呢（和目标检测中不太一样，请问这样有什么好处呢？）

对于进一步缩小，你指的是？
